pkg/krmtotf/resource.go

```diff
@@ -79,6 +79,15 @@ func NewResource(u *unstructured.Unstructured, sm *corekccv1alpha1.ServiceMappin
 
 func NewResourceFromResourceConfig(rc *corekccv1alpha1.ResourceConfig, p *tfschema.Provider) (*Resource, error) {
 	tfResource, ok := p.ResourcesMap[rc.Name]
+	// Pure Direct Resource does not have ResourceMap.
+	if rc.Direct {
```

Comment:
I am likely missing a trick.  Are we going to have to set `Direct` somewhere?  Can it be set from the direct registry?

Reply:
This config here is to distinguish a Direct resource that does not need to have any resource config. The direct registry is more for controller, to determine whether to use a Direct Controller or TF/DCL controller when it has multiple options.